grin/src/Grin/Syntax/Extended.hs

```diff
@@ -0,0 +1,149 @@
+{-# LANGUAGE DeriveDataTypeable, DeriveGeneric, DeriveAnyClass, DeriveFunctor, TypeFamilies #-}
+{-# LANGUAGE DeriveFoldable, DeriveTraversable, PatternSynonyms #-}
+{-# LANGUAGE TemplateHaskell, StandaloneDeriving, OverloadedStrings #-}
+{-# LANGUAGE LambdaCase #-}
+module Grin.Syntax.Extended
+  ( module Grin.Syntax.Extended
+  , module Grin.SyntaxDefs
+
+  -- reexports from Grin.Syntax
+  , Ty(..)
+  , ExternalKind(..)
+  , External(..)
+  , Lit(..)
+  , CPat(..)
+  , _CPatNodeTag
+  , _TyCon
+  , _TyVar
+  , _TySimple
+  , _CPatLit
+  , _CPatDefault
+  , isExternalName
+  ) where
+
+import Data.Data
+import Control.DeepSeq
+import Data.Binary
+import Data.Functor.Foldable.TH
+import Data.Int
+import Data.Text (Text, isPrefixOf)
+import Data.Vector
+import Data.Word
+import GHC.Generics (Generic)
+import Lens.Micro.Platform
+import qualified Data.ByteString.Short as B
+
+import Grin.SyntaxDefs
+import Grin.TypeEnvDefs
+
+import Grin.Syntax
+  ( Ty(..)
+  , ExternalKind(..)
+  , External(..)
+  , Lit(..)
+  , CPat(..)
+  , _CPatNodeTag
+  , _TyCon
+  , _TyVar
+  , _TySimple
+  , _CPatLit
+  , _CPatDefault
+  , isExternalName
+  )
+
+-- * GRIN Value
+
+data Val
+  -- CHANGE: Name
```

Comment:
What do you mean by "CHANGE" here?

Reply:
I don't think the difference between the old and new syntax has to be noted in the source. That's what the issue and git are about.

Reply:
Alright, I can remove these.

Reply:
Done.